README.md

```diff
@@ -188,3 +188,6 @@ Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Con
 
 ## Contact
 You can reach the admins, maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or intereact with the community. Join Us!
+
+## License
+Open Source Programs Backend is licensed under the GNU General Public License v3.0. To know more about it you can read [GPL v3](LICENSE) guidelines.
```

Comment:
link should be renamed to `License`. Also remove the word `guidelines` as it is a License and doesn't contains guidelines.

Reply:
As the file i am referring through the link is named  ` LICENSE `  not ` License ` so if i will change it ,it will not point to it .
Ok i will remove the word `guidelines`.

Reply:
@Subhangini I meant this :
`[License](LICENSE)`

Reply:
ok ok